### PR TITLE
Parse tags from markdown import

### PR DESCRIPTION
**Pull request by ocavue**
Parse hash tags from the markdown and returns them as an array of strings. 

See the test for all edge cases I can think of when parsing tags.

### CODE REVIEW

File: src/helpers/markdown/plugins/parse-tags.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import {buildTagUrl} from '../../tag'
+
+type ParseTagOptions = {
+  graphId: string
+  linkHost: string
+}
+
+// Don't match any URL reserved character https://en.wikipedia.org/wiki/URL_encoding
+const TAG_REGEX = /#([^\s!#$&'"*+,\\/:;=?@%|(){}[\]<>]+)/giu
```
Comment (maccman) [inline src/helpers/markdown/plugins/parse-tags.ts:13]: Can we simplify this and use `\w`

Comment (ocavue) [reply src/helpers/markdown/plugins/parse-tags.ts:13]: `\w` won't be able to match other language characters, like `#中文` or `#Русский`. I'm not sure how to simplify it if we want to have an ok i18n support.

**Review by maccman — APPROVED** [review]
